### PR TITLE
Prepping the release branch

### PR DESCRIPTION
**Pull request by rchiodo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,104 @@
...
 
+## 2022.5.100 (7 June 2022)
+
+### Enhancements
+
+1. Document context keys for keybinding 'when' clauses.
+   ([#6573](https://github.com/Microsoft/vscode-jupyter/issues/6573))
+1. Alert boxes of the form `<div style="alert alert-danger">` are now styled as colored boxes, to match how they are in Jupyter.
+   (thanks [Eric Wieser](https://github.com/eric-wieser/))
+   ([#8399](https://github.com/Microsoft/vscode-jupyter/issues/8399))
```
Comment (sadasant) [inline CHANGELOG.md:11]: This issue isn't closed 🤔

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,104 @@
...
+1. Add telemetry test to verify we output a specific set for different operations.
+   ([#6883](https://github.com/Microsoft/vscode-jupyter/issues/6883))
+1. Added a Performance test to test and ensure expected performance characteristics.
+   ([#7437](https://github.com/Microsoft/vscode-jupyter/issues/7437))
+1. Don't skip code completions when the server is busy. Instead let it timeout if the server doesn't come back.
+   ([#9797](https://github.com/Microsoft/vscode-jupyter/issues/9797))
+1. Add test for remote https jupyter servers.
+   ([#9844](https://github.com/Microsoft/vscode-jupyter/issues/9844))
+1. Create test for turning on old debugger tests and create a test for deleting tmp test files.
+   ([#10041](https://github.com/Microsoft/vscode-jupyter/issues/10041))
```
Comment (sadasant) [inline CHANGELOG.md:79]: This issue isn't closed, and it has a regression it seems.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,104 @@
...
+   ([#9826](https://github.com/Microsoft/vscode-jupyter/issues/9826))
+1. Allow usage of the jupyter API in stable builds so Juptyer Power Toys can use it.
+   ([#9868](https://github.com/Microsoft/vscode-jupyter/issues/9868))
+1. Support notebook debugging in the web extension.
+   ([#9973](https://github.com/Microsoft/vscode-jupyter/issues/9973))
+1. Support widgets that can be downloaded from a CDN in the web extension. Non CDN widgets will come later.
+   ([#9984](https://github.com/Microsoft/vscode-jupyter/issues/9984))
+1. Add editor context key support into the web extension.
+   ([#9990](https://github.com/Microsoft/vscode-jupyter/issues/9990))
+1. Fix problem with PYTHONNOUSERSITE being set even when not desired. There's a setting now that will set this environment variable on kernel launch if it's needed: 'jupyter.excludeUserSitePackages'.
```
Comment (sadasant) [inline CHANGELOG.md:41]:
```suggestion
1. Fix problem with `PYTHONNOUSERSITE` being set even when not desired. There's a setting now that will set this environment variable on kernel launch if it's needed: `jupyter.excludeUserSitePackages`.
```

how about these quotes?

Comment (rchiodo) [reply CHANGELOG.md]: Will do

File: CHANGELOG.md (hunk 1 above)
Comment (rchiodo) [reply CHANGELOG.md:11]: It should have been. I closed it.

File: CHANGELOG.md (hunk 2 above)
Comment (rchiodo) [reply CHANGELOG.md]: I'll remove it from the changelog.

**Review by sadasant — APPROVED** [review]
Looks good to me!

**Pull request merged**